content/en/real_user_monitoring/mobile_and_tv_monitoring/integrated_libraries/flutter.md

```diff
@@ -171,6 +134,93 @@ final routerDelegate = BeamerDelegate(
 );
 ```
 
+## Webview Tracking
+
+Real User Monitoring allows you to monitor web views and eliminate blind spots in your hybrid mobile applications.
+
+The Datadog Flutter SDK has packages for working with both [`webview_flutter`][8] and [`flutter_inappwebview`][9]. For more information, refer to the [Web View Tracking documentation page][10].
+
+## gRPC
+
+Datadog provides [`datadog_grpc_interceptor`][6] for use with the [grpc Flutter package][7]. The gRPC interceptor automaticly tracks gRPC requests as RUM Resources and enables distributed tracing with APM.
+
+### Setup
+
+Add `datadog_grpc_interceptor` to your `pubspec.yaml` or by running `flutter pub add datadog_grpc_interceptor` from your terminal:
+
+```yaml
+dependencies:
+  # Other dependencies
+  datadog_grpc_interceptor: ^1.1.0
+```
+
+To inter this plugin, create an instance of `DatadogGrpcInterceptor`, then pass it to your generated gRPC client:
+
+```dart
+import 'package:datadog_grpc_interceptor/datadog_grpc_interceptor.dart'
+
+// Initialize Datadog, be sure to set the [DatadogConfiguration.firstPartyHosts] member
+// to enable Datadog Distributed Tracing
+final config = DatadogConfiguration(
+  // ...
+  firstParthHosts = ['localhost']
+)
+
+// Create the gRPC channel
+final channel = ClientChannel(
+  'localhost',
+  port: 50051,
+  options: ChannelOptions(
+    // ...
+  ),
+);
+
+// Create the gRPC interceptor with the supported channel
+final datadogInterceptor = DatadogGrpcInterceptor(DatadogSdk.instance, channel);
+
+// Create the gRPC client, passing in the Datadog interceptor
+final stub = GreeterClient(channel, interceptors: [datadogInterceptor]);
+```
+
+## GraphQL (gql_link)
+
+Datadog provides [`datadog_gql_link`][1] for use with most GraphQL Flutter libraries, including `graphql_flutter` and `ferry`. The link automatically tracks GraphQL requests as RUM resources, adds query names, mutation names, and variables as attributes of the Resource, and enables distributed tracing in APM.
+
+### Setup
+
+Add `datadog_gql_link` to your `pubspec.yaml` or by running `flutter pub add datadog_gql_link` from your terminal:
+
+```yaml
+dependencies:
+  # Other dependencies
+  datadog_gql_link: ^1.0.0
+```
+
+When creating your GraphQL link, add the `DatadogGqlLink` above your terminating link. For example:
+
+```dart
+final graphQlUrl = "https://example.com/graphql";
+
+final link = Link.from([
+  DatadogGqlLink(DatadogSdk.instance, Uri.parse(graphQlUrl)),
+  HttpLink(graphQlUrl),
+]);
+```
+
+If you are tracking non-GraphQL network calls with `datadog_tracking_http_client`, you need to configure the tracking plugin to ignore requests to your GraphQL endpoint. Otherwise, GraphQL resources will be reported twice, and APM traces may be broken. Ignore your GraphQL endpoint by using the `ignoreUrlPatterns` parameter added to `datadog_tracking_http_client` version 2.1.0.
```

Comment:
```suggestion
If you are tracking non-GraphQL network calls with `datadog_tracking_http_client`, you need to configure the tracking plugin to ignore requests to your GraphQL endpoint. Otherwise, GraphQL resources are reported twice, and APM traces may be broken. Ignore your GraphQL endpoint by using the `ignoreUrlPatterns` parameter added to `datadog_tracking_http_client` version 2.1.0.
```